compose/cli/docker_client.py

```diff
@@ -32,4 +32,4 @@ def docker_client():
         )
 
     timeout = int(os.environ.get('DOCKER_CLIENT_TIMEOUT', 60))
-    return Client(base_url=base_url, tls=tls_config, version='1.18', timeout=timeout)
+    return Client(base_url=base_url, tls=tls_config, version='1.19', timeout=timeout)
```

Comment:
I've been using `auto` here when I write things with `docker-py` (https://github.com/docker/docker-py/blob/master/docker/clientbase.py#L63). I know we've talked about this before.

If we change this to `1.19` everyone is forced to update to the latest dockerd even if they don't use the one minor feature that requires it.  Some users are going to want to upgrade to the latest docker-compose to pickup bug fixes and new features, without upgrading dockerd immediately.

I think it would be good to either use `auto` or support it with #1157